catalog/firebase.json

```diff
@@ -10,7 +10,7 @@
         "destination": "/index.html"
       }
     ],
-    "ignore": ["**/.*"],
+    "ignore": ["**/.*", "robots.txt"],
```

Comment:
I'm guessing that this is the way to do it, but I don't know how to test it without actually deploying it?

Reply:
I can easily deploy to stage and test there.